nntrainer/layers/common_properties.h

```diff
@@ -1075,6 +1075,19 @@ class BNPARAMS_BETA_INIT final : public EnumProperty<InitializerInfo> {
   static constexpr const char *key = "beta_initializer";
 };
 
+/**
+ * @brief RMS_NORM_GAMMA_INIT Initialization Enumeration Information
+ */
+class RMS_NORM_GAMMA_INIT final : public EnumProperty<InitializerInfo> {
+public:
+  /**
+   * @brief Construct a RMS_NORM_GAMMA_INIT object
+   */
+  RMS_NORM_GAMMA_INIT(Initializer value = Initializer::ONES) { set(value); };
+  using prop_tag = enum_class_prop_tag;
+  static constexpr const char *key = "gamma_initializer";
```

Comment:
It seems that there is a duplication of key value ("gamma_initializer") for BNPARAMS_GAMMA_INIT.
How about changing the BNPARAMS_GAMMA_INIT class to GammaInitializer and using it in common?

Reply:
Thank you! That's my bad.  It must be critical if it wasn't fixed. I will update it.

Reply:
I'm thinking of changing all BNPARAMS_{*}_INIT parameter into {*}Initializer for generalized update.
However, I'm not sure it is okay in terms of backward compatibility.